content/en/docs/devops-user-guide/examples/a-maven-project.md

```diff
@@ -101,27 +99,29 @@ In this example, all workloads are deployed in `kubesphere-sample-dev`. You must
            DOCKERHUB_NAMESPACE = 'Docker Hub Namespace'
            APP_NAME = 'devops-java-sample'
```

Comment:
How about change the app name to `devops-maven-sample`? Because I found that our target repository is changed to <https://github.com/kubesphere/devops-maven-sample.git>.